lib/application/pages/home/collections/update/update_collection_memos.dart

```diff
@@ -48,6 +48,7 @@ class UpdateCollectionMemos extends HookConsumerWidget {
             key: ValueKey(metadata),
             pageIndex: index,
             metadata: metadata,
+            onUpdate: (memoMetadata) => vm.updateMemoAtIndex(index, metadata: metadata),
```

Comment:
This wrong reference was fixed in the next PR. Please ignore.

```suggestion
            onUpdate: (memoMetadata) => vm.updateMemoAtIndex(index, metadata: memoMetadata),
```